request/tests/test-request.js

```diff
@@ -438,7 +438,7 @@ o.spec("xhr", function() {
 		o("set timeout to xhr instance", function() {
 			mock.$defineRoutes({
 				"GET /item": function() {
-					return {status: 200, responseText: ''}
+					return {status: 200, responseText: ""}
```

Comment:
Remove this from the PR please. I like the consistency increase but would rather keep this PR as focused as possible and touching as few files as possible.

Reply:
Of course. Asking so I can do as the Romans do: Do you all normally amend commits and force push to update PRs in this situation, or just revert and add commits?

Reply:
Whatever strikes your fancy, I'm gonna do a squash onto `next` at the end anyways.